modules/reindex/src/main/java/org/elasticsearch/index/reindex/AbstractAsyncBulkByScrollAction.java

```diff
@@ -129,57 +128,44 @@ public void start() {
             return;
         }
         try {
-            // Default to sorting by _doc if it hasn't been changed.
-            if (firstSearchRequest.source().sorts() == null) {
-                firstSearchRequest.source().sort(fieldSort("_doc"));
-            }
             startTime.set(System.nanoTime());
-            if (logger.isDebugEnabled()) {
-                logger.debug("executing initial scroll against {}{}",
-                        firstSearchRequest.indices() == null || firstSearchRequest.indices().length == 0 ? "all indices"
-                                : firstSearchRequest.indices(),
-                        firstSearchRequest.types() == null || firstSearchRequest.types().length == 0 ? ""
-                                : firstSearchRequest.types());
-            }
+            scrollSource.start(response -> {
+                logger.debug("[{}] documents match query", response.getTotalHits());
```

Comment:
The log debug message is kinda duplicate with the one in `ScrollableHitSource`, no?